src/github.ts

```diff
@@ -65,7 +65,9 @@ export class GitHub {
   }
 
   static get workflowRunURL(): string {
-    return `${GitHub.serverURL}/${github.context.repo.owner}/${github.context.repo.repo}/actions/runs/${github.context.runId}`;
+    const runID = process.env.GITHUB_RUN_ID || github.context.runId;
+    const runAttempt = process.env.GITHUB_RUN_ATTEMPT || 1;
```

Comment:
Fyi "run attempt" is not yet part of github context but will be in a future release of `@actions/github` package: https://github.com/actions/toolkit/commit/faa425440f86f9c16587a19dfb59491253a2c92a

Keep that in mind for follow-up.